src/DocBlock/Annotation.php

```diff
@@ -217,9 +217,10 @@ public function getTypes()
      */
     public function setTypes(array $types)
     {
-        $pattern = '/'.preg_quote($this->getTypesContent()).'/';
-
-        $this->lines[0]->setContent(preg_replace($pattern, implode('|', $types), $this->lines[0]->getContent(), 1));
+        $this->lines[0]->setContent(
```

Comment:
I think it would be better if you format it like this:
```php
$this->lines[0]->setContent(
    preg_replace(
        '/'.preg_quote($this->getTypesContent(), '/').'/',
        implode('|', $types),
        $this->lines[0]->getContent(),
        1
    )
);
```
Or just change the variable `$pattern`